google/cloud/bigtable/tests/instance_admin_emulator.cc

```diff
@@ -48,6 +48,14 @@ class InstanceAdminEmulator final
       auto contents = bigtable::internal::make_unique<google::protobuf::Any>();
       contents->PackFrom(stored_instance);
       response->set_allocated_response(contents.release());
+
+      // Add cluster into clusters_
+      for (auto& kv : request->clusters()) {
```

Comment:
See my comments in https://github.com/GoogleCloudPlatform/google-cloud-cpp/pull/696#discussion_r194236910

Reply:
Do not worry about this, I added #697 to track the optimization.  Please fix the conflicts and the problems running against production though.